packages/cli/src/commands/generate/script/templates/script.js.template

```diff
@@ -0,0 +1,8 @@
+// To access your database
+import { db } from '../api/src/lib/db'
```

Comment:
```suggestion
import { db } from '$api/src/lib/db'
```

Reply:
I intentionally didn't do this, because you don't get typescript autocomplete if we do this (and not sure if its worth adding tsconfig and maintaining it for scripts)